### DISCUSSION TITLE
MLFlow Tracking Server Outside Databricks

### DISCUSSION THREAD

**Discussion by debanshu-mutt** [mlflow/mlflow#7074] [Q&A] [answered]
Hi, so I have been trying to access Mlflow tracking server outside of databricks following this documentation 
https://docs.databricks.com/mlflow/access-hosted-tracking-server.html#step-3-configure-the-mlflow-cli
The authorization is done but when I try to create a new experiment or call an old experiment file 
I get this error 
 File "C:\Users\DMU108\Anaconda3\envs\mlflow_env\lib\site-packages\mlflow\utils\rest_utils.py", line 206, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/experiments/create failed with error code 404 != 200. Response body: ''

Can you please help out in this?

**Comment by JHibbard** [comment]
Hi debanshu-mutt,

I need more details to help:

-  are you using a REST API token or a credentials file?
- Have you been able to successfully access other resources of the remote server to verify your authorization is setup correctly?
- It'd also be great to get a code snippet or more details on how you're creating the experiment.

Thanks!
Jim

**Comment by debanshu-mutt** [comment, marked as answer]
Hi Jim, I rechecked and found out that that there was some connection issue with the token. The issue is resolved now, thanks!

**Comment by harupy** [comment]
@debanshu-mutt Can we close #7075?
